builtin/files/userdata/cloud-config-worker

```diff
@@ -387,7 +382,8 @@ coreos:
         {{- end }}
         $KUBELET_OPTS"
         Restart=always
-        RestartSec=10
+        RestartSec=10        
+
```

Comment:
nit: extra new line

Reply:
I actually added this purposefully to more clearly separate the `[Service]` and `[Install]` block similar to other unit definitions here. 

We should probably unify on one in another pull request (extra line between unit blocks, or without extra line), what are your thoughts?

Reply:
Sorry for the late response. I'm ok with the extra line but maybe would be much cleared if

```suggestion
        RestartSec=10        

{{/* END of kubelet [Service] */}}
```

Reply:
I'm happy with that, I pushed those changes up.